x-pack/legacy/plugins/reporting/public/components/report_info_button.tsx

```diff
@@ -266,7 +266,6 @@ export class ReportInfoButton extends Component<Props, State> {
           info: null,
           error: kfetchError,
         });
-        throw kfetchError;
```

Comment:
@tsullivan I've taken the liberty to remove this re-throw. As far as I can see, this just results in an unhandled promise rejection error - at least in the tests, so I assume in prod as well. And since we set the error state in the line above, I think we're ok by not re-throwing. But I'm not 100% sure, and since it seems that you touched this code last, I thought I'd just ping you if you don't mind.

Reply:
+1 it looks like we are already doing what we need to with this error, which is add it to the component state so it can appear in a message.